### ISSUE TITLE
CI / Checks (pull_request) on Github Actions failing with isort 5.3.2 -> 5.4.0

### ISSUE THREAD

**Issue by simonjayhawkins** [pandas-dev/pandas#35703]
```
2020-08-13T10:30:51.9470350Z isort --version-number
2020-08-13T10:30:52.2249999Z 5.4.0
2020-08-13T10:30:52.2443133Z Check import format using isort
2020-08-13T10:30:56.4886273Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/_testing.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4892079Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/_typing.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4892907Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/tests/frame/test_analytics.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4893331Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/tests/extension/base/__init__.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4893737Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/tseries/frequencies.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4894183Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/tseries/offsets.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4894928Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/tseries/holiday.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4895326Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/plotting/_matplotlib/timeseries.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4895747Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/index.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4896139Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/common.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4896552Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/ops/__init__.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4897157Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/window/__init__.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4897613Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/dtypes/dtypes.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4898032Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/dtypes/common.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4898422Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/reshape/melt.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4898832Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/reshape/merge.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4899228Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/internals/__init__.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4899642Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/core/tools/datetimes.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4900046Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/api/__init__.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4900461Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/util/__init__.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4900859Z ##[error]ERROR: /home/runner/work/pandas/pandas/pandas/io/pytables.py Imports are incorrectly sorted and/or formatted.
2020-08-13T10:30:56.4901145Z Check import format using isort DONE
```

**Comment by simonjayhawkins** [comment]
looks like an issue with isort removing #noqa directives so may need to pin for now.

**Comment by VirosaLi** [comment]
Isort's iteration is too fast for a linting/style package.

**Comment by jreback** [comment]
hmm can we modify the .isort.cfg to handle this?

**Comment by simonjayhawkins** [comment]
closing as fixed in 5.4.1 timothycrosley/isort#1381

**Comment by timothycrosley** [comment]
@VirosaLi, I agree!  I'm looking to improve the stability of the release schedule going forward, and would appreciate feedback on what cadence would be most fitting - to provide a good balance between stability and not leaving users with their desired features waiting finished in master for too long.
